cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java

```diff
@@ -205,6 +205,8 @@ public MasterServiceMain() {
     login(cConf);
 
     Configuration hConf = HBaseConfiguration.create();
+    hConf.set(Constants.HBase.CLIENT_RETRIES, cConf.get(Constants.HBase.MASTER_CLIENT_RETRIES));
```

Comment:
Makes sense. Thanks for the comments. Addressed comments and also move the tephra constants from `AbstractProgramTwillRunnable` to `AbstractDistributedProgramRunner`